blscurve.nimble

```diff
@@ -45,23 +45,40 @@ task test, "Run all tests":
   # Internal SHA256
   test "-d:BLS_FORCE_BACKEND=blst", "tests/blst_sha256.nim"
 
+  # batch verification
+  test "-d:BLS_FORCE_BACKEND=blst", "tests/t_batch_verifier.nim"
+
+  # No OpenMP in default Clang on Mac
```

Comment:
But (also apropos the gcc 4.4 comment elsewhere), https://www.openmp.org/wp-content/uploads/OpenMPOpt-in-LLVM-SC20-JD.pdf suggests that LLVM in general has OpenMP support?

Reply:
No, Apple LLVM doesn't have OpenMP support either. They want everyone to use Grand Central Dispatch. And unlike Windows who also wants everyone to use Windows Fibers, MSVC OpenMP uses uses Windows Fibers underneath while Apple just doesn't provide anything.